package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@xstate/viz",
+  "name": "xstate-viz-app",
```

Comment:
This is internal-only, we won't publish it in the future as this, right?

Reply:
right - i have just changed it to something more distinct, so it shouldnt conflict now with a potential `@xstate/viz` package or smth